solr/core/src/java/org/apache/solr/cluster/events/impl/CollectionsRepairEventListener.java

```diff
@@ -76,10 +78,12 @@
   private int waitForSecond = DEFAULT_WAIT_FOR_SEC;
 
   private ScheduledThreadPoolExecutor waitForExecutor;
+  private PlacementPluginFactory<? extends MapWriter> placementPluginFactory;
```

Comment:
Should this rather be `<? extends PlacementPluginConfig>`?

Reply:
Yes.